drivers/scheduler/rke/rke.go

```diff
@@ -184,6 +198,104 @@ func (r *Rancher) GetProjectID(projectName string) (string, error) {
 	return projectId, fmt.Errorf("no project matching the given projectName %s was found", projectName)
 }
 
+//CreateUsersForRancherProject Creates rancher users based on the supplied number and adds them to the project
+func (r *Rancher) CreateUsersForRancherProject(projectName string, numberOfUsers int) ([]string, error) {
```

Comment:
You can break this helper function down to smaller chunks.
First create a simple function `CreateUserForRancherProject(username string, project string)`
Then create a function `CreateMultipleUsersForRancherProject((projectName string, userIDList []string)` and call `CreateUserForRancherProject` inside it.

We should not have `RandomString` func in `rke.go`. If you want to create random users. Then you build an array with random usernames in the test case itself and pass that array to the function `CreateMultipleUsersForRancherProject`. Or loop the array and call `CreateUserForRancherProject` in the test case

Reply:
Thank you for the comment @mkoppal-px , I have addressed this comment in the latest commit. Please review.